src/lib/FlightTasks/FlightTasks.hpp

```diff
@@ -103,6 +103,18 @@ class FlightTasks
 	 */
 	const vehicle_constraints_s getConstraints();
 
+	/**
+	 * Get task avoidance desired waypoints
+	 * @return auto triplets in the mc_pos_control
+	 */
+	const vehicle_trajectory_waypoint_s getAvoidanceWaypoint();
```

Comment:
Probably we should switch all these getters to work by reference in the future. One `vehicle_trajectory_waypoint_s` on a 32-Bit platform is already >240Byte, this can explode in resource usage.